packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -52,8 +52,19 @@ import {
 	TelemetryDataTag,
 } from "@fluidframework/telemetry-utils";
 
-import { IGCRuntimeOptions, RuntimeHeaders } from "./containerRuntime";
-import { getSummaryForDatastores } from "./dataStores";
+import { IGCRuntimeOptions, RuntimeHeaders } from "../containerRuntime";
+import { getSummaryForDatastores } from "../dataStores";
+import {
+	getGCVersion,
```

Comment:
The names of these imply there's still some mixing - should some of these live in the gc folder too?

Reply:
Maybe yeah. I am working on refactoring summarizer files next. Will refactor these next - https://dev.azure.com/fluidframework/internal/_workitems/edit/3461